Geometry/CommonTopologies/interface/HGCDiskGeomDet.h

```diff
@@ -0,0 +1,35 @@
+#ifndef RecoHGCal_HGCTracking_HGCDiskGeomDet_h
+#define RecoHGCal_HGCTracking_HGCDiskGeomDet_h
+
+// (mmatthew) Wrapper class for GeomDet corresponding to one layer of HGCal. Used by PatternRecognitionByKF.
+
+#include "Geometry/CommonDetUnit/interface/GeomDet.h"
```

Comment:
Not important, but more correct:
```suggestion
#include "Geometry/CommonTopologies/interface/GeomDet.h"
```